scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -225,6 +227,8 @@ task GatherVcfs {
     Int memory_mb = 7500
     Int disk_size_gb = ceil(3*size(input_vcfs, "GiB")) + 500
   }
+
+  File monitoring_script = "gs://gvs_quickstart_storage/cromwell_monitoring_script.sh"
```

Comment:
nit: do we want to keep the monitoring script in here?

Reply:
Do you mean in the wdl, or where I have it stored (in gs://gvs_quickstart_storage/ ?)

Reply:
I mean, do we always want to monitor this task when we run this WDL?

Reply:
👍 Probably not.